modules/support_ticket/support_ticket.module

```diff
@@ -253,6 +244,143 @@ function support_ticket_load($nid = NULL, $reset = FALSE) {
 }
 
 /**
+ * Determines what fields have changed when the support ticket is updated.
```

Comment:
Should say:

/**
* Implements hook_entity_update()
*/
function support_ticket_entity_update($support_ticket) {
  // Determines what fields have changed when the support ticket is updated.